composer.json

```diff
@@ -14,6 +14,7 @@
         }
     ],
     "require": {
+        "php": "^5.5 || ^7.0",
```

Comment:
Ah yes, I was confused. Actually I was wondering about the difference between `||` and `|`, which I see much more commonly, e.g. https://github.com/symfony/symfony/blob/master/composer.json#L21